bin/format_coverage.dart

```diff
@@ -220,7 +220,8 @@ Environment parseArgs(List<String> arguments) {
 /// are contained by it if it is a directory, or a [List] containing the file if
 /// it is a file.
 List<File> filesToProcess(String absPath) {
-  final filePattern = RegExp(r'^dart-cov-\d+-\d+.json$');
+  print(absPath);
+  final filePattern = RegExp(r'^.*\.json$');
```

Comment:
Add a simple test for this changed behavior?

Reply:
going to remove this commit and send another PR w/ something better here